views/ministryteams.ejs

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html lang="en">
+  
+<%- include widgets/header.ejs %>
+<%- include widgets/mainNav.ejs %>
+<div class="overlay" id="our-beliefs">
```

Comment:
Could also change banner image to something unique. But this could also be another task for another newer web team member.

Reply:
Should we do this before we merge, or can it wait until after? If after I'm in favor of assigning it to a newer web team member because there's not many tasks for people to do right now to get familiar with the website. If before it'll be faster if I do it.

Reply:
We can do it after

Reply:
Can I close this and merge? or are we waiting for something?